alembic/versions/2025_06_05_1200-c3323aaa8b4c_add_http_request_columns_to_workflow_run_blocks.py

```diff
@@ -0,0 +1,30 @@
+"""add http_request block columns to workflow_run_blocks
```

Comment:
remove this alembic file. i don't think we've done the migration for these columns in the main